crud/select/plan.lua

```diff
@@ -206,6 +212,16 @@ function select_plan.new(space, conditions, opts)
 
     local sharding_index = primary_index -- XXX: only sharding by primary key is supported
 
+    if scan_after_tuple ~= nil then
+        if scan_iter == box.index.LE then
+            scan_iter = box.index.LT
+        elseif scan_iter == box.index.EQ then
+            scan_iter = box.index.GE
+        elseif scan_iter == box.index.REQ then
+            scan_iter = box.index.LT
+        end
+    end
```

Comment:
nit: I think this change should be done before `local sharding_index = primary_index` line. 

`local sharding_index = primary_index` and `local sharding_key ...` are in "sharding" group of code.